packages/cli/test/charm-operations-integration.test.ts

```diff
@@ -0,0 +1,75 @@
+import { describe, it } from "@std/testing/bdd";
```

Comment:
I think we should just drop these tests, superfluous. We should prefer to extend the `integration.sh` script for more robust testing.